models.py

```diff
@@ -584,10 +567,6 @@ def format_for_template(self, version=None):
           'text': STANDARDIZATION[self.standardization],
           'val': d.pop('standardization', None),
         },
-        'footprint': {
```

Comment:
Does the comma on the previous line need to go away?

Reply:
Nope. Trailing commas are considered good in Python because it allows for addition and removal of other dictionary items with a cleaner-looking diff (like the one in this CL).
https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas
and
https://google.github.io/styleguide/pyguide.html